neo4j_indexer.py

```
@@ -0,0 +1,1454 @@
+    extraction_results: list[ExtractionResult],
+):
+    """문서/청크/엔티티를 Neo4j에 저장"""
+    driver = get_driver()
+    async with driver.session() as session:
+        # 문서 노드 생성
+        await session.run(
+            "MERGE (d:Document {id: $id}) SET d.filename = $filename, d.chunk_count = $count",
+            id=doc_id, filename=filename, count=len(chunks),
+        )
+
+        for chunk, extraction in zip(chunks, extraction_results):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)